src/less/navbar.less

```diff
@@ -17,7 +17,7 @@
   &:extend(.clearfix all);
 
   @media (min-width: @grid-float-breakpoint) {
-    border-radius: @navbar-border-radius;
+    .border-radius(@navbar-border-radius);
```

Comment:
Seems like we should use autoprefixer. We can specify which browsers to support and it will figure out if it needs to add browser prefixes. It also cleans outdated prefixes!

_
Nathan Young

On Jul 15, 2016, 6:20 AM -0700, Matthew Osborn <notifications@github.com>, wrote:

In src/less/navbar.less<https://github.com/CenturyLinkCloud/Cyclops/pull/59#discussion_r70971037>:


> @@ -17,7 +17,7 @@
>    &:extend(.clearfix all);
>
>    @media (min-width: @grid-float-breakpoint) {
> -    border-radius: @navbar-border-radius;
> +    .border-radius(@navbar-border-radius);


Personally I'm indifferent, the mixins came from bootstrap so at the time it was the "fast" way because there was no work :)

Given our support matrix I'm not sure that we really need to prefix much, maybe just a couple of the newest properties...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/CenturyLinkCloud/Cyclops/pull/59/files/9eec78ec184603010b3ed808f8f4699961ac6eeb#r70971037>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABYVqzRD4VFFGGBXJTbdG2Sf9cr247aWks5qV4kCgaJpZM4JM4YE>.